### ISSUE TITLE
[RFE] Template for grabing test suite

### ISSUE THREAD

**Issue by voxik** [fedora-ruby/gem2rpm#102]
Quite lot of gems don't include their test suite. It would be nice to detect such packages and try to provide some boilerplate in the template.

**Comment by pvalena** [comment]
You're right. I'm adding those lines into almost any new package I'm creating.
